geetools/ee_dictionary.py

```diff
@@ -80,3 +82,42 @@ def getMany(self, list: list | ee.List) -> ee.List:
                 d.geetools.getMany(["foo", "bar"]).getInfo()
         """
         return ee.List(list).map(lambda key: self._obj.get(key))
+
+    def toTable(self, valueType: ee.List | ee.Dictionary | Any = Any) -> ee.FeatureCollection:
+        """Convert a ee.Dictionary to a ee.FeatureCollection with no geometries (table).
+
+        The keys will always be stored in `system:index` column.
```

Comment:
The long description needs to be improved so the user can understand exactly what this function is doing with both the keys and the values. In the current implementation, the default behaviour (1 feature, non-collection dType) is not described.